drivers/serial/serial_shell.c

```diff
@@ -0,0 +1,157 @@
+/* Copyright (C) 2024 Bang & Olufsen A/S, Denmark
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/uart.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(serial_shell, CONFIG_UART_LOG_LEVEL);
+
+#define ARGV_DEV     1
+#define ARGV_TX_DATA 1
```

Comment:
If these offsets refer to the input argument list, shouldn't their indexes differ?

Reply:
They're not used in the same context. So might  not make sense to use defines.. Think I'm gonna remove them instead :+1:

Reply:
Hm, they're being used multiple times (in their own context) so I think I'm gonna keep them. Can add a comment on why they're both 1 instead